### ISSUE TITLE
egress gateway integration tests are failing postsubmits on release-1.5 branch

### ISSUE THREAD

**Issue by dgn** [istio/istio#21576]
See https://prow.istio.io/view/gcs/istio-prow/logs/integ-k8s-114_istio_release-1.5_postsubmit/181 for an example.

They were introduced by https://github.com/istio/istio/pull/21535.

fyi @yangminzhu

**Comment by yangminzhu** [comment]
I think the root cause is the postsubmits install istio via helm and doesn't include egress gateway:

```
2020-02-27T00:49:10.383853Z	info	ci	Checking pods ready...
2020-02-27T00:49:10.412993Z	info	ci	  [ 0]                  istio-galley-866bd4fdb-tdgtp         Pending (Pending)
2020-02-27T00:49:10.413031Z	info	ci	  [ 1]         istio-ingressgateway-5f7cd6987b-48rr2         Pending (Pending)
2020-02-27T00:49:10.413038Z	info	ci	  [ 2]                  istio-pilot-5684bb9cb5-m7r9r         Pending (Pending)
2020-02-27T00:49:10.413043Z	info	ci	  [ 3]                 istio-policy-54ccfbdf8c-h52r8         Pending (Pending)
2020-02-27T00:49:10.413048Z	info	ci	  [ 4]                 istio-policy-54ccfbdf8c-kswqv         Pending (Pending)
2020-02-27T00:49:10.413052Z	info	ci	  [ 5] istio-security-post-install-istio-testing-xh4xv         Pending (Pending)
2020-02-27T00:49:10.413056Z	info	ci	  [ 6]               istio-telemetry-9fbd656dc-nspld         Pending (Pending)
2020-02-27T00:49:10.413060Z	info	ci	  [ 7]                   prometheus-74d8b55f54-9jqr8         Pending (Pending)
2020-02-27T00:49:20.413223Z	info	ci	Checking pods ready...
```

I think I may fix the failure by adding the egress gateway but I'm not sure is it expected to use helm in postsubmit? (presubmit has changed to use operator) @howardjohn

**Comment by howardjohn** [comment]
It is expected. Its questionable whether its a good idea but thats how its currently structured for 1.5 (on master helm is removed)

**Comment by yangminzhu** [comment]
@howardjohn Thanks for the confirm, I will fix it in release-1.5.

@dgn Thanks for the report, I have sent out the fix.

**Comment by dgn** [comment]
@yangminzhu thank you very much. I can confirm the tests pass now
